src/lib/db/addon-store.ts

```diff
@@ -16,6 +16,7 @@ const COLUMNS = [
     'events',
     'projects',
     'environments',
+    'created_at',
```

Comment:
This was missing from our return value. We claimed that we returned it, but we never did. Seems like an oversight.